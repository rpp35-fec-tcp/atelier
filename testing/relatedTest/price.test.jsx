/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Price from '../../client/src/components/relatedItems&comparison/price';
import fakeDataStyle from './fakeDataStyle';

describe('Price Component', () => {
  it('snapshot test', () => {
    const modal = renderer.create(<Price defaultItem={fakeDataStyle[3]} />).toJSON();
    expect(modal).toMatchSnapshot();
  });
  it('snapshot test', () => {
    const modal = renderer.create(<Price defaultItem={fakeDataStyle[2]} />).toJSON();
    expect(modal).toMatchSnapshot();
  });
});
