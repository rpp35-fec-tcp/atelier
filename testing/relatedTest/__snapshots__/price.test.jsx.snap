// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price Component snapshot test 1`] = `
<div>
  <p
    className="originalPrice"
  >
    69.00
  </p>
</div>
`;

exports[`Price Component snapshot test 2`] = `
<div>
  <p
    className="crossOutOriginalPrice"
  >
    69.00
  </p>
  <p
    className="salePrice"
  >
    35.00
  </p>
</div>
`;
